packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -114,14 +114,20 @@ export type PendingMessageResubmitData = Pick<
 	runtimeOp: LocalContainerRuntimeMessage;
 };
 
+export interface PendingBatchResubmitMetadata extends BatchResubmitInfo {
```

Comment:
it seemed better to bundle all of these arguments together to align with #24516, though it does add a little more churn from my previous PR (sorry!)